src/ontology/uberon-edit.obo

```diff
@@ -224112,6 +224113,15 @@ relationship: part_of UBERON:0001769 ! iris
 property_value: curator_notes "In urodeles, lens regeneration proceeds through two steps. The first step involves the entire circumference of the iris pigmented epithelium. The second step is confined to the pigmented epithelial cells located along the dorsal rim of the iris and involves dorsal-specific activation of the Wnt signaling pathway." xsd:string
 property_value: dcterms-date "2023-06-06T08:06:04Z" xsd:dateTime
 
+[Term]
+id: UBERON:8480064
+name: secretory coil of eccrine sweat gland
+def: "Tightly coiled tubule whose function is the production of sweat. The secretory tubule has three types of cells: clear, dark, and myoepithelial. The myoepithelial cells are contractile cells that provide structural support to the gland against hydrostatic pressure generated during sweat production. The secretory cells can be classified into clear cells and dark cells based on their affinities to basic dyes and granule contents. The clear cells are without secretory granules and are responsible for the secretion of primary sweat. The dark cells have abundant dark granules in the cytoplasm and are thought to regulate the function of the clear cells." [PMID:31608304, PMID:34395417]
```

Comment:
Consider moving **cell type** details to the CL terms.

Reply:
Consider adding axioms to capture these cell types, e.g., 'has part' some 'clear cell of eccrine sweat gland'.

Reply:
@bvarner-ebi I tried to add 'has part' some 'clear cell of eccrine sweat gland' as "SubClass Of" but I can't. Where these axioms must be added?